front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/BaseCarrousel/ScrollableCarrousel/index.tsx

```diff
@@ -99,66 +93,63 @@ const ScrollableCarrousel = ({
-          </>
-        )}
-        {!isSmallerThanPhone && (
-          <>
-            {showNextButton && (
-              <ScrollButton
-                variant="right"
-                top={`${scrollButtonTop}px`}
-                onClick={() => {
-                  if (!scrollContainerRef) return;
-                  scrollContainerRef.scrollLeft += cardWidth + CARD_GAP;
-                }}
-                onMouseEnter={() => setMouseOverNextButton(true)}
-                onMouseLeave={() => {
-                  setMouseOverNextButton(false);
-                  if (nextButtonShouldDisappearAfterMouseMove) {
-                    setShowNextButton(false);
-                    setNextButtonShouldDisappearAfterMouseMove(false);
-                  }
-                }}
-              />
-            )}
-            {showNextGradient && <Gradient variant="right" />}
-          </>
-        )}
-      </CarrouselContainer>
+      <SkipButton onSkip={() => skipCarrousel(endId)} />
+      <HorizontalScroll
+        setRef={(ref) => {
+          if (ref) {
+            setScrollContainerRef(ref);
+            handleButtonVisiblity(ref, hasMore);
+          }
+        }}
+      >
+        {children}
+      </HorizontalScroll>
+      {!isSmallerThanPhone && (
+        <>
+          {showPreviousButton && (
+            <ScrollButton
+              variant="left"
+              top={`${scrollButtonTop}px`}
+              onClick={() => {
+                if (!scrollContainerRef) return;
+                scrollContainerRef.scrollLeft -= cardWidth + CARD_GAP;
+              }}
+              onMouseEnter={() => setMouseOverPreviousButton(true)}
+              onMouseLeave={() => {
+                setMouseOverPreviousButton(false);
+                if (previousButtonShouldDisappearAfterMouseMove) {
+                  setShowPreviousButton(false);
+                  setPreviousButtonShouldDisappearAfterMouseMove(false);
+                }
+              }}
+            />
+          )}
+          {showPreviousGradient && <Gradient variant="left" />}
+        </>
+      )}
+      {!isSmallerThanPhone && (
+        <>
+          {showNextButton && (
+            <ScrollButton
+              variant="right"
+              top={`${scrollButtonTop}px`}
+              onClick={() => {
+                if (!scrollContainerRef) return;
+                scrollContainerRef.scrollLeft += cardWidth + CARD_GAP;
+              }}
+              onMouseEnter={() => setMouseOverNextButton(true)}
+              onMouseLeave={() => {
+                setMouseOverNextButton(false);
+                if (nextButtonShouldDisappearAfterMouseMove) {
+                  setShowNextButton(false);
+                  setNextButtonShouldDisappearAfterMouseMove(false);
+                }
+              }}
+            />
+          )}
+          {showNextGradient && <Gradient variant="right" />}
+        </>
+      )}
       <i id={endId} />
```

Comment:
SkipButton still seems to work fine and doesn't need to move up together with CarrouselContainer.